src/pages/angular/config.md

```diff
@@ -66,6 +64,90 @@ This will set the default text for `ion-back-button` to `Go Back`. However, if y
 
 In this example we have used our `ion-back-button` in such a way that the text can be dynamically updated if there were to be a change that warranted it, such as a language or locale change. The `getBackButtonText` method would be responsible for returning the correct text.
 
+## Per-Platform Config
+
+Ionic Config can also be set on a per-platform basis. For example, this allows you to disable animations if the app is being run in a browser on a potentially slower device. Developers can take advantage of the Platform utilities to accomplish this.
+
+Since the config is set at runtime, you will not have access to the Platform Dependency Injection. Instead, you can use the underlying functions that it uses directly.
+
+```typescript
+import { isPlatform, IonicModule } from '@ionic/angular';
+
+@NgModule({
+  ...
+  imports: [
+    BrowserModule,
+    IonicModule.forRoot({
+      animated: !isPlatform('mobileweb')
+    }),
+    AppRoutingModule
+  ],
+  ...
+})
+```
+
+The `isPlatform()` call returns `true` or `false` based upon the platform that is based in. See the [Platform Documentation](./platform#platforms) for a list of possible values.
+
+In the example above, we are disabling all animations in our Ionic app only if the app is running in a mobile web browser. 
+
+```typescript
+import { isPlatform, IonicModule } from '@ionic/angular';
+
+const getConfig = () => {
+  if (isPlatform('hybrid')) {
+    return {
+      backButtonText: 'Previous',
+      tabButtonLayout: 'label-hide'
+    }
+  }
+  
+  return {
+    menuIcon: 'ellipsis-vertical'
+  }
+}
+@NgModule({
+  ...
+  imports: [
+    BrowserModule,
+    IonicModule.forRoot(getConfig()),
+    AppRoutingModule
+  ],
+  ...
+})
+```
+
+This example allows you to set an entirely different configuration based upon the platform, falling back to a default config if no platforms match.
```

Comment:
I think it would be less confusing if you added what the example does prior to showing it. I thought this was talking about the example after it at first.